Source/Core/Core/DSP/Jit/DSPJitUtil.cpp

```diff
@@ -25,7 +25,7 @@ void DSPEmitter::dsp_reg_stack_push(int stack_reg)
 	MOV(16, R(tmp1), M(&g_dsp.r.st[stack_reg]));
 	MOVZX(64, 8, RAX, R(AL));
 	MOV(16, MComplex(EAX, EAX, SCALE_1,
-			 PtrOffset(&g_dsp.reg_stack[stack_reg][0],nullptr)), R(tmp1));
+			 PtrOffset(&g_dsp.reg_stack[stack_reg][0], static_cast<u16*>(nullptr))), R(tmp1));
```

Comment:
nullptr it s of type std::nullptr_t, it needs to be explicitly converted in templated contexts otherwise deduction will fail (which is, IMO, a good thing)